src/eip4824.ts

```diff
@@ -35,28 +35,54 @@ router.get('/:space', async (req, res) => {
 
 router.get('/:space/members', async (req, res) => {
   let space: any = {};
+  let uniqueVotersResult: any = {};
 
   try {
-    space = await getSpace(req.params.space);
+    const spaceId = req.params.space;
+    const cursor = req.query.cursor || null;
+    // Default pageSize is set to 100, this is the query limit.
 
+    space = await getSpace(spaceId);
     if (!space.verified) return res.status(400).json({ error: 'INVALID' });
+
+    uniqueVotersResult = await getUniqueVotersForSpace(spaceId, cursor);
   } catch (e) {
     return res.status(404).json({ error: 'NOT_FOUND' });
   }
 
-  const members = [...space.admins, ...space.moderators, ...space.members].map(
-    member => ({
-      type: 'EthereumAddress',
-      id: member
-    })
-  );
-
-  return res.json({
+  // Combine all members and map them to JSON strings for uniqueness check
+  const allMembersJSON = [
+    ...space.admins.map(admin =>
+      JSON.stringify({ type: 'EthereumAddress', id: admin })
```

Comment:
You don't need to add the type and id object format as string here, this can be done in the end once you have the final list instead.